Package/msi/Msi.Packaging.targets

```diff
@@ -0,0 +1,13 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project xmlns="http://schemas.microsoft.com/developer/msbuild/2003" ToolsVersion="4.0">
+  <Target Name="MsiPackage" Condition="'$(OS)' != 'Unix' " AfterTargets="Package">
+    <ItemGroup>
+      <CleanupFiles Include="$(StageDir)\net45\IronPythonTest.dll"/>
+    </ItemGroup>
+    <Delete Files="@(CleanupFiles)" />
```

Comment:
Should we move this delete to the `Package` target if it's useful for all package targets?

Reply:
I think it's only needed for Msi because of the way it specifies files to include. The other packages allow specifying excludes and such, but the Wix stuff doesn't seem to.

Reply:
Maybe I missed it, but I'm not seeing it excluded in the Debian package.

Reply:
https://github.com/IronLanguages/ironpython2/pull/416/files#diff-ea9b249df442099c197f0858e7f580caR16